src/components/Swap/Summary/Details.tsx

```diff
@@ -27,61 +51,109 @@ interface DetailProps {
 
 function Detail({ label, value, color }: DetailProps) {
   return (
-    <ThemedText.Caption userSelect>
+    <ThemedText.Body2 userSelect>
       <Row gap={2}>
-        <span>{label}</span>
+        <Label>{label}</Label>
         <Value color={color}>{value}</Value>
       </Row>
-    </ThemedText.Caption>
+    </ThemedText.Body2>
+  )
+}
+
+interface AmountProps {
+  tooltipText?: string
+  label: string
+  amount: CurrencyAmount<Currency>
+  usdcAmount?: CurrencyAmount<Currency>
+}
+
+function Amount({ tooltipText, label, amount, usdcAmount }: AmountProps) {
+  const width = useWidgetWidth()
+
+  let formattedAmount = formatCurrencyAmount(amount, NumberType.TokenTx)
+  if (formattedAmount.length > 9) {
+    formattedAmount =
+      width >= 420
+        ? formatCurrencyAmount(amount, NumberType.SwapTradeAmount)
+        : formatCurrencyAmount(amount, NumberType.TokenNonTx)
+  }
+
+  return (
+    <Row flex align="space-between">
```

Comment:
i don't think `space-between` is a valid value for the `align-items` attribute. are you sure this is having any effect?

Reply:
also, space-between is the default value for the `justify` prop, so you may not need this

Reply:
good to know, ty